src/logger.js

```diff
@@ -7,14 +7,20 @@
 
 import util from 'util';
 import winston from 'winston';
+import { getConfig } from './settings';
+/** @import { Configuration } from './settings' */
```

Comment:
Should we remove this?

Reply:
This is not a comment, it is a JSDoc import so we can use the `Configuration` defined as a `@typedef` in `src/settings.js`.

We can remove it, personally I just like having the argument types since it helps with development (LSP completion and checks) and it helps with documentation since it helps understand what is expected to be in the config object.